.eslintrc.yml

```diff
@@ -45,3 +45,11 @@ rules:
   # these avoid React is defined but not used error, and is set in plugin:react/recommended
   # react/jsx-uses-react: 2
   # react/react-in-jsx-scope: 2
+
+  react/jsx-filename-extension:
+    - 1
+    - extensions: [".js", ".jsx"]
+
+  jsx-a11y/anchor-is-valid:
```

Comment:
do we need to add this rule here? I can see airbnb eslint config already overwrites this rule: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react-a11y.js#L188

Reply:
But when I use `<Link to="/">` I still got the following lint error:
```
The href attribute is required on an anchor. Provide a valid, navigable address as the href value  (jsx-a11y/anchor-is-valid)
```

Reply:
i finally find the reason... it's only on master, not released to npm yet
https://github.com/airbnb/javascript/commit/4817a6b75cd44fe8b28a16b06d33a5c65603a74c